app/views/shared/fields/_note.html.erb

```diff
@@ -46,7 +46,7 @@
         elsif note_type != nil && !note_type.blank?
           notes[note_type.titleize] = note
         else
-          notes['General Note'] = note
+          notes['General Note'.to_s] = note
```

Comment:
Hi Matt,
I've changed it to use the Array data structure to process all the note types, which looks more straight forward for understanding. See https://github.com/ucsdlib/damspas/commit/3986ccf0b6f3f21f1553573c95d05b9867f1f448.